generator/main.js

```diff
@@ -1,12 +1,22 @@
-const { faker } = require("@faker-js/faker");
+const { faker } = require('@faker-js/faker');
+const { appendFileSync } = require('fs');
 
-const generateProduct = () => ({
-  name: faker.commerce.productName(),
-  department: faker.commerce.department(),
-  description: faker.commerce.productDescription(),
-  price: faker.commerce.price(),
-  image: faker.image.imageUrl(),
-  releasedAt: faker.date.past(),
-});
+const generateProduct = () => [
+  faker.commerce.productName(),
+  faker.commerce.department(),
+  faker.commerce.productDescription(),
+  faker.commerce.price(),
+  faker.image.imageUrl(),
+  faker.date.past(),
+];
 
-console.log(generateProduct()); // TODO: write to file
+const arrayToRow = (array) => {
+  const enquotedArray = array.map((el) => `"${el}"`);
+  return enquotedArray.join(',') + '\n';
+};
+
+for (let i = 0; i < 1000; i++) {
+  const product = generateProduct();
+  const row = arrayToRow(product);
+  appendFileSync('contents.csv', row);
```

Comment:
You should put the file into `data` directory and call it `products.csv`. The `data` directory is in the root directory, path from generator is `../data`.

Reply:
Fixed in the recent commit